src/applications/hca/tests/hca-helpers.js

```diff
@@ -386,6 +386,7 @@ function initEnrollmentStatusMock(token = null) {
       enrollmentDate: '2018-01-24T00:00:00.000-06:00',
       preferredFacility: '987 - CHEY6',
       parsedStatus: 'none_of_the_above',
+      e2eTestNotInMVI: true, // If true, sets isUserInMVI to false for e2e testing.
```

Comment:
@tlei123 I'm not entirely sure, but I think we can fake a 404 response from the `enrollment_status` endpoint using the `client.mockData()` function, similar to how it's done here (https://github.com/department-of-veterans-affairs/vets-website/blob/26f7edca07ee0b40fb7bc3df29c46ed5f2ffee50/src/platform/forms/tests/save-in-progress/01-sip-load-fail.e2e.spec.js#L77). Maybe you could add a new `initEnrollmentStatusMock404()` helper to this helpers file, that would use `client.mockData()` and call that at the start of the e2e test file? I really don't like adding the hack to the reducer just to make it possible to test the upload page in the form.

Reply:
Test now refactored to use mockData; reducer changes now taken out.